src/main/java/org/kainos/ea/db/DatabaseConnector.java

```diff
@@ -21,22 +20,26 @@ public static Connection getConnection() throws SQLException {
             Properties props = new Properties();
             props.load(propsStream);
 
-            user = props.getProperty("user");
-            password = props.getProperty("password");
-            host = props.getProperty("host");
-            name = props.getProperty("name");
+            String user = props.getProperty("user");
+            String password = props.getProperty("password");
+            String host = props.getProperty("host");
+            String name = props.getProperty("name");
 
             if (user == null || password == null || host == null) {
-                throw new IllegalArgumentException("Properties file must exist " +
+                throw new IllegalArgumentException("Properties file must exist "
+                        +
                         "and must contain user, password, name, and host properties");
             }
 
-            connection = DriverManager.getConnection("jdbc:mysql://" +
-                    host + "/" + name +
+            connection = DriverManager.getConnection("jdbc:mysql://"
+                    +
+                    host
+                    + "/"
+                    + name
+                    +
                     "?useSSL=false", user, password);
```

Comment:
Is the linter telling you to do this? I'd find this quite weird. I.e. + and host are on new lines, but + name are on the same line.

Reply:
Yeah that was the linter :)